.github/workflows/pre_release_version_bump.yml

```diff
@@ -0,0 +1,38 @@
+name: Manually Triggered Version Bumps
```

Comment:
@Jaybsoni Just out of curiosity, how do you trigger it manually? Does it create a button on github?

Reply:
Hey, yes you go under the actions tab and it should be titled 'Manually Triggered Version Bumps'. Upon selecting it, it will run